SafeSnap-Agents/Cooldown-Alert/src/agent.spec.ts

```diff
@@ -0,0 +1,290 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+  HandleBlock,
+  TransactionEvent,
+  BlockEvent,
+} from "forta-agent";
+import {
+  createAddress,
+  TestBlockEvent,
+  TestTransactionEvent,
+  encodeParameters,
+} from "forta-agent-tools";
```

Comment:
check the `runBlock` tool from our module and consider using it in your tests

Reply:
I have looked into it, and I don't think it'll work in my case. Saw the code is:
```
export async function runBlock(agent: Agent, block: BlockEvent, ...txns: TransactionEvent[]): Promise<Finding[]> {
  let findings: Finding[] = [];

  findings.push(...(await agent.handleBlock(block)));
  for (let tx of txns) findings.push(...(await agent.handleTransaction(tx)));

  return findings;
}
```
However, for my scenario, I need the handler to run `handleTransaction` before `handleBlock` because the txn handler is what populates the `questionIds` array, which the block handler then loops through.